dm/dm/master/server.go

```diff
@@ -572,6 +572,18 @@ func (s *Server) OperateTask(ctx context.Context, req *pb.OperateTaskRequest) (*
 
 	resp.Result = true
 	resp.Sources = s.getSourceRespsAfterOperation(ctx, req.Name, sources, []string{}, req)
+
+	if expect == pb.Stage_Stopped {
+		// delete meta data for optimist
```

Comment:
can you explian why `when the task stops, remove all metadata for optimism, when restarting the task, we can fetch table info from downstream instead.` what will happen when we not delete  meta here?

Reply:
etcd meta will become larger and larger.

Reply:
so we also better to do this with pessimist ?

Reply:
Optimistic may have some "Grayscale DDL" e.g. 
tb1: add column new_col1, tb2 add column new_col2. 
then both infos will be kept until they add another finally.
Pessimistic doesn't have this problem.